config/settings.yml

```diff
@@ -1329,6 +1329,7 @@ vanotify:
         form21_4142_confirmation_email: form21_4142_confirmation_email_template_id
         form21_4142_error_email: form21_4142_error_email_template_id
         form21_4142_received_email: form21_4142_received_email_template_id
+        form27_8832_action_needed_email: form27_8832_action_needed_email_template_id
```

Comment:
I see this was added to staging https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3192. Do any other envs need it?

Reply:
@rmtolmach at the moment only the staging environment template is setup. When production gets added I'll submit another PR with the new key.